docs/emulators/spd.md

```diff
@@ -0,0 +1,132 @@
+
+The emulator provides 3 ways to edit `.SPD` files:
+
+- [Sprite patching](#patching-sprites)
+- [Texture replacement](#replacing-textures)
+- [Adding new sprites](#adding-new-sprites) (advanced)
+
+## Patching Sprites
+
+Sprite patching in this context refers to the technique of appending a provided texture to the `.SPD`s texture dictionary, and patching a sprite entry to point to the new texture.
+
+After editing the texture containing the sprite you want to change, change the texture's filename to reflect the appropriate sprite id(s), which can be found using PersonaEditor. 
+
+The filename should be `spr_x.dds` where `x` is the sprite ids, using [Id Notation](#id-notation).
+
+Textures should be the same size as the originals, and the sprites should be in the same place, unless you're supplying an edited `.spdspr` file.
+
+If you are supplying an edited `.spdspr` file, the filename should be `spr_x.spdspr` (or `.sprt`) with `x` being the sprites id.
+
+!!! warning
+    
+    Moving around sprite positions may occasionally break sprite positioning. This has only been observed with P5R so far, and affects sprites that appear next to button prompts as well as a lot of the pause menu animation sprites. These sprites must be put into an image with the same size as the original with the sprites in the correct positions in order to be displayed correctly.
+
+### Examples
+
+- To edit sprite id 15, name the texture `spr_15.dds`.
+- To edit sprites 15 and 20, name the texture `spr_15_20.dds`
+- To edit sprites 15, 20, 25-30, and 45-55, name the texture `spr_15_20_25-30_45-55.dds`
+
+## Replacing Textures
+
+Under the hood, texture replacement is really just sprite patching, but affects every sprite that pointed to the original texture.
+
+To replace a texture, simply name your edited texture `tex_x.dds` where `x` is the id of the texture.
+
+You may also want to exclude some sprites from being affected by this method. You can do this by adding a tilda `~` at the end of the texture name, followed by the sprite ids in [Id Notation](#id-notation).
+
+### Example
+
+- To replace all sprites in texture id 4, name the texture `tex_4.dds`
+- To replace all sprites in texture id 4 except for sprite id 15-20, and 24, name the texture `tex_4~15-20_24.dds` 
+
+## Adding New Sprites
+
+Adding a new sprite works similarly to patching sprites, but requires that an accompanying sprite entry file be present.
+
+### Example
+
+To add a previously non-existent sprite id 420, create a sprite entry file named `spr_420.spdspr`, and name the texture `spr_420.dds`.
+You can add multiple new sprites by putting multiple sprite entry files, and naming the texture using [Id Notation](#id-notation).
+
+## Id Notation
+
+Both sprite patching and texture replacement allow for multiple sprites to be patched/excluded. sprite ranges are denoted by dashes `-`, and sprite ids and ranges are separated by underscores `_`.
```

Comment:
I'm gonna leave it as is, since the exclude filter isn't really part of the notation. It's only used by the texture 'replacement' function as a separator between the texture id, and the sprites to be excluded (which are in "Id notation")

Reply:
Alright, that works.